tensorbay/cli/log.py

```diff
@@ -342,21 +345,25 @@ def _add_graph_full(
         else:
             # Merge branches.
             del self._layer_colors[self._merge_pointer]
-            merge_prefixes = self._get_merge_prefixes()
+            merge_prefixes = self._get_merge_prefixes(self._merge_pointer, self._pointer)
             lines.extend(self._combine_details(list(merge_prefixes), list(splitlines)))
         return "".join(lines)
 
     def _get_colorful_prefixes(self) -> List[str]:
         return [click.style("|", fg=color) for color in self._layer_colors]
 
-    def _get_merge_prefixes(self) -> Iterator[str]:
+    def _get_merge_prefixes(self, merge_pointer: int, pointer: int) -> Iterator[str]:
         prefixes = []
         for color in self._layer_colors:
             prefixes.append(click.style("|", fg=color))
             prefixes.append(" ")
-        for i in range(self._merge_pointer, self._pointer, -1):  # type: ignore[arg-type]
+
+        for i in range(merge_pointer, pointer, -1):
```

Comment:
can we also use `enumerate` in the outer loop?

Reply:
In the outer loop, `i` is only used for indicating the position, so `enumerate` is not needed